File: SDKs/amazonaws.com/gamelift/2015-10-01/python/src/sdk/models/shared/creatematchmakingrulesetoutput.py
import dataclasses
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import matchmakingruleset as shared_matchmakingruleset


@dataclass_json
@dataclasses.dataclass
class CreateMatchmakingRuleSetOutput:
    r"""CreateMatchmakingRuleSetOutput
    Represents the returned data in response to a request operation.
    """
    
    rule_set: shared_matchmakingruleset.MatchmakingRuleSet = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('RuleSet') }})
    

File: SDKs/amazonaws.com/gamelift/2015-10-01/java/lib/src/main/java/openapisdk/models/shared/CreateMatchmakingRuleSetOutput.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * CreateMatchmakingRuleSetOutput
 * Represents the returned data in response to a request operation.
**/
public class CreateMatchmakingRuleSetOutput {
    @JsonProperty("RuleSet")
    public MatchmakingRuleSet ruleSet;
    public CreateMatchmakingRuleSetOutput withRuleSet(MatchmakingRuleSet ruleSet) {
        this.ruleSet = ruleSet;
        return this;
    }
}